app/assets/javascripts/scripts.js

```diff
@@ -219,3 +219,18 @@ function hasCheckedRadio(q_num) {
 function iAgree(){
   $("#disclaimer-submit")[0].disabled = !$("#accept")[0].checked;
 }
+
+/*
+ * ============================================================================
+ * Dashboard code
+ * ============================================================================
+ */
+  $(".button-collapse").sideNav({

```

Comment:
In the general script section of this file, there's already something similar to this. We may want to combine them, or make sure only one of the side nav code runs (either in admin or assessment). Because this code currently conflicts with the general scripts code. However, I think your code can just replace mine.
`$(".button-collapse").sideNav({
       edge: "right"
     });`